### ISSUE TITLE
How to convert negative numbers

### ISSUE THREAD

**Issue by nfisher86** [MicrosoftDocs/OfficeDocs-Support#941]
When I try to convert negative latitudes or longitudes the VB code does not return the correct value (I am entering the DMS in a separate cell).  So far I have tried including a ' and a space in front of the negative sign (otherwise excel thinks that I am entering a formula).  How do I edit the code or how should I enter negative latitudes and have the VB code return the correct values?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: be969d50-8518-365f-f3ed-5c0e14d55104
* Version Independent ID: 11bad482-781b-bbae-ce60-53f8164d137e
* Content: [Convert degrees/minutes/seconds angles to or from decimal angles - Office](https://docs.microsoft.com/en-us/office/troubleshoot/excel/convert-degrees-minutes-seconds-angles)
* Content Source: [Office/Client/excel/convert-degrees-minutes-seconds-angles.md](https://github.com/MicrosoftDocs/OfficeDocs-Support/blob/public/Office/Client/excel/convert-degrees-minutes-seconds-angles.md)
* Product: **office-perpetual-itpro**
* GitHub Login: @simonxjx
* Microsoft Alias: **v-six**

**Comment by simonxjx** [comment]
Ths code is updated. Thanks for your comment.

**Comment by nfisher86** [comment]
Thank you very much!

**Comment by nfisher86** [comment]
I'm sorry, but the code no longer works.  With the updated code I now have to enter "10° 27 ' 36"" to get the code to work...before I entered "10° 27' 36"".  And I still am unable to accurately convert negative latitudes or longitudes.  I have tried entering a "W" for West longitudes and a "-" and neither work.  What convention is the code looking for to work for negative numbers?  And why do I now have to enter a space in front of the minute sign whereas previously I didn't have to?

**Comment by nfisher86** [comment]
For instance, "-10° 27' 36"" should return a result of -10.46.  But right now it will return a result of -9.54.  In order to get that number the code must be doing the calculation of "=-10+(27/60)+(36/3600)"...which is wrong.  It should be doing "=-10-(27/60)-(36/3600)".  Perhaps an If block or something to look for a "W" (for West longitude) or "S" (for South latitude)?  Or to just look for a negative sign at the beginning?

**Comment by nfisher86** [comment]
And in the Convert_Decimal code, I think lines 12 and 13 still need to be: 

    InStr(1, Degree_Deg, "'") - InStr(1, Degree_Deg, _
    "°") - 2)) / 60

**Comment by JaimeBee** [comment]
I have a similar problem using the code to convert DD to DMS. I entered a negative Latitude in DD and got a DMS one degree larger (larger if talking in absolute values) and in general different in M and S than if I enter it without the negative symbol:

-33.121            :    Number entered
-34° 52 ' 44"    :    Output
  33° 7 ' 16"      :    Output when negative symbol deleted

So I think the code for DD to DMS is not working well. I used the output when the negative symbol in the input is ignored and just manually added the negative symbol to the result. Let me know if you fix the code.
